pokecli.py

```diff
@@ -116,6 +116,7 @@ def get_commit_hash():
 
                 while True:
                     bot.tick()
+                    inventory.refresh_inventory()
```

Comment:
The mobile app contact server every 10s as know as the heart, we should not change the api sequence. Even now, we have much work to refactor the call sequence.